### PR TITLE
Changes to Redbird.swift to make support for pub/sub redis

### PR DESCRIPTION
**Pull request by surendratiwari3**
i have added a function that is supported when you require pub/sub with redis to function well

### CODE REVIEW

File: Sources/Redbird/Redbird.swift [hunk 1]
```diff
@@ -127,6 +127,18 @@ public class Redbird {
...
+    //added by surendra tiwari to support subcribe
+    public func read() throws -> RespObject
+    {
+        var ret: RespObject?
+        try self.handleComms {
+            let reader: SocketReader = self.socket
+            let (responseObject, _) = try InitialParser().parse([], reader: reader)
+                ret = responseObject
+        }
+        return ret!
```
Comment (tannernelson) [inline Sources/Redbird/Redbird.swift]: Can we throw an error instead of implicitly unwrapping?

**Comment by surendratiwari3** [comment]
let client = try Redbird()
let client_ip = try Redbird(config: RedbirdConfig(address: "127.0.0.1"))

let client1 = try Redbird()
let client_ip1 = try Redbird(config: RedbirdConfig(address: "127.0.0.1"))

do {
try client.command("SUBSCRIBE", params: ["hello"])

    print("Sending 'PING' to Redis server at \(client1.address):\(client1.port)")
    let response11 = try client1.command("PING", params: []).toString()
    print("Response: \(response11)")

    while true
    {
            let response2 = try client.read().toArray()
            var count = 0
            for element in response2
            {
                    count = count+1
                    let content = (element as! RespBulkString).content
                    if count==3
                    {
                            //published key
                            print(content)
                            //sending the command to redis
                            print("Sending HMGET \(content) to Redis server at \(client1.address):\(client1.port)")
                            let balance_res = try client1.command("HMGET", params: ["\(content)", "balance"]).toArray()
                            //get the balance from the redis
                            for ebal in balance_res
                            {
                                    let bal = (ebal as! RespBulkString).content
                                    print("Response: \(bal)")
                            }
                    }
            }
    }
} catch {
print("Encountered error (error)")
fatalError("(error)")
}

please have a look . i open two connection one for subscription and another for other request to redis.

File: Sources/Redbird/Redbird.swift (hunk 1 above)
Comment (surendratiwari3) [inline Sources/Redbird/Redbird.swift]: will you explore more?

**Comment by tannernelson** [comment]
This has been fixed.

**Pull request closed without merging**